src/StorageMover/StorageMover.Autorest/custom/UpdateAzStorageMoverAzSmbFileShareEndpoint.ps1

```diff
@@ -53,9 +53,10 @@ INPUTOBJECT<IStorageMoverIdentity>: Identity Parameter
 .Link
 https://learn.microsoft.com/powershell/module/az.storagemover/update-azstoragemoversmbfileshareendpoint
 #>
-function Update-AzStorageMoverSmbFileShareEndpoint {
+function Update-AzStorageMoverAzSmbFileShareEndpoint {
     [OutputType([Microsoft.Azure.PowerShell.Cmdlets.StorageMover.Models.Api20231001.IEndpoint])]
     [CmdletBinding(DefaultParameterSetName = 'UpdateExpanded', PositionalBinding =$false, SupportsShouldProcess, ConfirmImpact = 'Medium')]
+    [Alias("Update-AzStorageMoverSmbFileShareEndpoint")]
```

Comment:
We might should add a test case to make sure the old name still work.

Reply:
test case added